src/modules/elements/action-button/style.pcss

```diff
@@ -59,6 +59,6 @@
 .tribe-editor__action-button--icon-left {
 
 	& > svg {
-		margin-right: 6px;
+		margin-right: 5px;
```

Comment:
Spacer var? See comment above on "close but not quite" numbers

Reply:
using tec spacers var now